### PR TITLE
Some sample encodings for perfMedium

### PR DESCRIPTION
**Pull request by riedde**
During the discussion of the `<perfMedium>` proposal at metadata-ig meetings, some sample encodings were captured and are now available here as snippets. These are not abstract contents, but the encoded representation of existing cases from the music literature. These are meant to serve as a pool of ideas and are intended to help other users.

### CODE REVIEW

File: MEI_4.0/Header/PerfMedium/organ/2_performers/Strauss_Johann-Jun_ValsForOrganPedal.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://music-encoding.org/schema/4.0.1/mei-all_anyStart.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://music-encoding.org/schema/4.0.1/mei-all_anyStart.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<perfMedium xmlns="http://www.music-encoding.org/ns/mei">
+   <head>Strauß, Johann (jun.): Vals for organ pedal (for 2 performers/4 feet)</head>
```
Comment (musicEnfanthen) [inline MEI_4.0/Header/PerfMedium/organ/2_performers/Strauss_Johann-Jun_ValsForOrganPedal.xml]: Waltz (?)

Comment (riedde) [reply MEI_4.0/Header/PerfMedium/organ/2_performers/Strauss_Johann-Jun_ValsForOrganPedal.xml]: Of course!

**Comment by musicEnfanthen** [comment]
@kepper @bwbohl Can we go forward with this?

**Review by bwbohl — APPROVED** [review]
validated against schema, no objections

**Pull request merged**